internals/onepassword/cli_v2.go

```diff
@@ -0,0 +1,137 @@
+package onepassword
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type OPV2CLI struct {
+	version string
+	isV2    bool
```

Comment:
same as above:
```go
type OPV2CLI struct {
	version string
}

func (op *OPV2CLI) IsV2() bool {
	return true
}
```

Reply:
Indeed it can be simplified. Nice catch 😄